osu.Game.Tests/Visual/TestCaseTabControl.cs

```diff
@@ -37,6 +47,21 @@ public TestCaseTabControl()
             {
                 text.Text = "Currently Selected: " + newFilter.ToString();
             };
+
+            AddStep("set selection to first tab", () => filter.Current.Value = filter.VisibleItems.First());
```

Comment:
Nitpick, but could you make these shorter? The last ones don't even fit on the buttons, and you could really just put "select first", "select next" etc.

Reply:
I guess?